nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -229,6 +241,23 @@ def _permuted_ols_on_chunk(scores_original_data, tested_vars, target_vars,
         scores_as_ranks_part += (h0_fmax_part[i].reshape((-1, 1))
                                  < scores_original_data.T)
 
+        if verbose > 0:
+            step = 11 - min(verbose, 10)
```

Comment:
This is almost funny.

Reply:
This is how it is implemented in searchlight. :) 
I can change that if needed...